src/components/resumes/ResumeList/Box.css

```diff
@@ -0,0 +1,15 @@
+.Name {
```

Comment:
Please remove the ResumeList folder, check how it is the projects folder structure.
You'll end up with something like this: 
```
/resumes
   /Resumes.tsx
   /Resumes.css
   /Box.tsx
   /Box.css
```

Reply:
change the file name from ResumeList to Resumes